Dockerfile

```diff
@@ -1,7 +1,22 @@
-FROM registry.opensource.zalan.do/stups/node:6.9-cd32
+FROM registry.opensource.zalan.do/library/ubuntu-18.04:latest
+
 
 MAINTAINER Zalando SE
 
+ENV NODE_VERSION 6.9.4
+
+RUN apt update && \
+    apt install -y --no-install-recommends curl ca-certificates && \
+    curl "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.gz" --output "node-v${NODE_VERSION}-linux-x64.tar.gz" && \
+    tar xzf node-v${NODE_VERSION}-linux-x64.tar.gz -C /opt && \
+    ln -s /opt/node-v${NODE_VERSION}-linux-x64/bin/node /usr/local/bin/node && \
+    ln -s /opt/node-v${NODE_VERSION}-linux-x64/bin/npm /usr/local/bin/npm && \
```

Comment:
I personally don't see npm used in the Dockerfile. Am i missing something ?
```suggestion
```

Reply:
It is in line no. 32. A `npm install` is done before setting the Dockerfile's `CMD` as `node yourturn.js`.

Reply:
My bad, I missed that

Reply:
No worries :)